### DISCUSSION TITLE
Household survey data

### DISCUSSION THREAD

**Discussion by jdebacker** [EAPD-DRB/OG-PHL#17] [Q&A]
The Philippine Statistical Authority has [a nice website where one can download the micro data](https://psada.psa.gov.ph/catalog/Household-based_Surveys/about) from various household surveys.  

What surveys would be helpful to calibrating OG-PHL?

Note some of the following needs:
* Hours worked (with this we can calibrate the $\chi^n_s$ to match the lifecycle profile of labor supply)
* Measures of personal income tax liability by household income (useful for calibrating personal income tax functions)
* Measures of net worth/wealth (from this we can adjust the $\chi^b$ and $\beta_j$ parameters of household preferences to generate a distribution of savings that matches this wealth distribution
* Measures of the receipt of bequests (useful for calibrating the $\zeta$ matrix that allocates aggregate bequests by age and household lifetime income group)
* Measures of the receipt of remittences (useful for calibrating parameters of the size and distribution of remittences)
* Measures of the receipt of government (non-pension) transfers (useful for calibrating the $\eta$ matrix that allocates transfers by age and household lifetime income group)

There might be some sources for some of this information and different ones for others.  Suggestions?

**Comment by ptuano** [comment]
The merged [Family Income and Expenditure Survey- Labor Force Survey] (https://psada.psa.gov.ph/catalog/FIESLFS/about) addresses many of the data needs: hours worked, income tax payments, remittances and, to some extent, non-pension government transfers. Unfortunately, I don't think measures of net worth and bequests are included in the merged dataset, unless others would know?
